settings_default.py

```diff
@@ -28,6 +28,8 @@
 
 # PRESIDENT
 PRESIDENT_ROLES_ID = 0 # Role id of president
+FIBRE_ID = 260861701279318019 #Id of user FibreTigre. The information is public
```

Comment:
Pour que le code reste le plus générique possible, je trouve ça mieux de laisser `0` ou, dans la fonction `clap`, mettre `ctx.guild.owner.id`.

Reply:
Changed to guild owner and remove from settings